libraries/AP_Logger/LogStructure.h

```diff
@@ -951,6 +951,28 @@ struct PACKED log_Esc {
     int16_t motor_temp;
 };
 
+struct PACKED log_CSRV {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;     
+    uint8_t id;
+    float position;
+    float force;
+    float speed;
+    uint8_t power_pct;
+};
+
+struct PACKED log_CESC {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;     
+    uint8_t id;
+    uint32_t error_count;
```

Comment:
Do we really need to burn this many bytes?  I'm assuming this is a high-rate message.